person.rb

```diff
@@ -8,14 +8,14 @@ class Person
   def initialize(age:, name: 'Unknown', parent_permission: true)
```

Comment:
- [ ] as the previous CR mentioned int line 8 you passed in an age parameter to your initialize constructor function like this age:, doing this translates to age: age meaning you are setting a default value with the variable name age but there is no where in your code that age is defined as as such this is an error in your code. 
the advise therefore is to remove the colon after age, this will fix this issue.

Reply:
I see, thank you